python/tk_multi_demo/demos/view_item_delegate/demo.py

```diff
@@ -401,6 +401,10 @@ def _expand_row_height_to_text_changed(self, state):
         Callback triggered by the checkbox to signal the ViewItemDelegate to expand each row to fit all
         item text (vertically).
         """
+        # For PySide6 compatibility, convert state to CheckState enum if not already
+        # an instance. Issue described at: https://forum.qt.io/post/743017
+        if not isinstance(state, QtCore.Qt.CheckState):
+            state = QtCore.Qt.CheckState(state)
```

Comment:
I would really love to patch this in tk-core, Norberto had looked at it in this [PR](https://github.com/shotgunsoftware/tk-core/pull/952) but did not end up getting it to work. If we have time to look into this, that would be awesome so that we do not have to add this check every time we use this QCheckBox signal/slot

Reply:
We can try again once we clean up the patchers and finally remove the QT4 interface.